release-notes/SupportedFeatures-v1.0.0.md

```diff
@@ -0,0 +1,65 @@
+|              |Kerberos/HTTP          |:white_check_mark:                   |:white_check_mark:                  |[:no_entry_sign:](#known_issues)           | [:no_entry_sign:](#known_issues)    |
+|              |Certificate/TCP       |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|              |Certificate/HTTP      |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :grey_question:                     |
+|Service Authentication|Certificate    |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|              |SPNIdentity/HTTP       |:white_check_mark:                   |:white_check_mark:                  |[:warning:](#known_issues)                 | [:warning:](#known_issues)          |
+|              |SPNIdentity/TCP        |[:warning:](#known_issues)           |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|              |UPNIdentity    |[:warning:](#known_issues)          |[:warning:](#known_issues)          |[:warning:](#known_issues)                  | [:warning:](#known_issues)          |
+|Client types  |ChannelFactory\<T\>    |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|              |ChannelBase\<T\>       |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|Extensibility |IClientMessageInspector|:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|              |IClientMessageFormatter|:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|Management    |ETW Tracing            |[:no_entry_sign:](#known_issues)      |:white_check_mark:                  |[:no_entry_sign:](#known_issues)          |[:no_entry_sign:](#known_issues)     |          
+
+
+#### UWP
+&ensp;&ensp;For more details about what UWP is please see this [dotnet Glossary doc](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/glossary.md).
+
+#### All Other Platforms
+&ensp;&ensp;Linux OSs includes centos, debian, fedora, linux, opensuse, rhel and ubuntu.
+
+&ensp;&ensp;For supported versions of each platform and other details please see the [.NET Core 1.0.0 Release Notes](https://github.com/dotnet/core/blob/master/release-notes/1.0/Release-Notes-1.0.0.md).
+
+#### Known_Issues
```

Comment:
I would suggest we don't include this section. Instead, we point the link back to the known issues section of our release notes.